src/pybind/mgr/balancer/module.py

```diff
@@ -189,6 +191,8 @@ def calc_stats(self, count, target, total):
             stddev = math.sqrt(dev / float(max(num - 1, 1)))
             score = score / max(sum_weight, 1)
             r[t] = {
+                'max': max(count[t].iteritems(), key=lambda x:x[1])[1],
```

Comment:
Can we fold the min/max calculation into the `for k, v in six.iteritems(count[t]):` loop, which looks more straightforward and efficient..

Reply:
or max(count[t].itervalues()) should suffice...

Reply:
> or max(count[t].itervalues()) should suffice...

done @xiexingguo